### DISCUSSION TITLE
permission problem when i push

### DISCUSSION THREAD

**Discussion by alaoui2001** [geeksblabla/blanat#62] [Q&A]
remote: Permission to geeksblabla/blanat.git denied to alaoui2001.
fatal: unable to access 'https://github.com/geeksblabla/blanat.git/': The requested URL returned error: 403

**Comment by chermehdi** [comment]
Hello, you are trying to push directly to `geeksblabla/blanat` repository, you should fork the repository and push a new branch in your own fork and create a PR to the `geeksblalba/blanat` repository.
